modules/manage/pages/kubernetes/troubleshooting/k-troubleshoot.adoc

```diff
@@ -72,40 +94,132 @@ spec:
       logLevel: debug
 ----
 
-```bash
+Then, apply this configuration:
+
+[source,bash]
+----
 kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
-```
+----
 --
 
 Helm::
 +
 --
+Choose between using a custom values file or setting values directly:
 [tabs]
 ====
 --values::
 +
+Specify logging settings in `logging.yaml`, then upgrade:
++
 .`logging.yaml`
-[,yaml]
+[source,yaml]
 ----
 logging:
   logLevel: debug
 ----
 +
-```bash
+[source,bash]
+----
 helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
   --values logging.yaml --reuse-values
-```
-
+----
 --set::
 +
-```bash
+Directly set the log level during upgrade:
++
+[source,bash]
+----
 helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
   --set logging.logLevel=debug
-```
+----
 ====
 --
 ======
 
+After applying these changes, verify the log level by <<view-redpanda-logs, checking the initial output of the logs>> for the Redpanda Pods.
+
+=== Set log levels for Redpanda subsystems
+
+You can specify log levels for individual subsystems, such as `rpc` and `kafka`, for more detailed logging control.
```

Comment:
do we have reference on all redpanda subsystems?

Reply:
This is something I think we need to cover in https://github.com/redpanda-data/documentation-private/issues/2333

Also submitted an issue to fix in core: https://github.com/redpanda-data/redpanda/issues/17711